src/ApiHandler.ts

```diff
@@ -607,8 +610,7 @@ export default class ApiHandler {
 		}
 	}
 
-	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-	async submitTx(extrinsic: string) {
+	async submitTx(extrinsic: string): Promise<{ hash: Hash }> {
```

Comment:
But this does not always return this data type.

Reply:
I think it always _returns_ this data type. Our errors are of a different format, but (from my understanding at least) since they are throws we do not include them in the data type